sagemaker-debugger/tensorflow_action_on_rule/detect_stalled_training_job_and_stop.ipynb

```diff
@@ -0,0 +1,581 @@
+{
```

Comment:
<p>Is there a reason why we can't use sagemaker.get_execution_role().. If "Admin" role is required for stopping the training, then we should document it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1301/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>